scenes/game/game.tscn

```diff
@@ -396,10 +398,10 @@ anchor_left = 1.0
 anchor_top = 1.0
 anchor_right = 1.0
 anchor_bottom = 1.0
-offset_left = 1060.0
-offset_top = 15.0
-offset_right = 1185.0
-offset_bottom = 190.0
+offset_left = 1058.0
+offset_top = 1.0
```

Comment:
This is the only way you moved significantly I think (from 15 to 1)?
Does this put the deck right on the edge of the screen?

Reply:
the opponent's deck is pretty close to the top now

![image](https://github.com/daniel-k-taylor/exceedgameclient/assets/17868497/b1ef11a2-ae5a-4a38-a683-452f4ea82984)